fuuk/people/models/person.py

```diff
@@ -153,3 +153,22 @@ def full_name(self):
             self.last_name.strip(),
             suffix and u", %s" % suffix or u"",
         )
+
+
+class AbstractAuthor(models.Model):
+    # abstract base class for author
+    first_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+
+    class Meta:
+        abstract = True
+
+
+class AbstractFullAuthor(AbstractAuthor):
+    # abstract base class for authors with prefix and suffix
```

Comment:
* Abstract author models should be probably in separate module.
 * Description of a class should be in docstring.